_pages/team.html

```diff
@@ -152,7 +152,104 @@ <h5>Ricardo Quiroz </h5>
             <p class="label">🖥️ Programmer 🖥️</p>
             <h5>Brandon Werbel</h5>
             <p>Brandon is a new member of the team and is a programmer.</p>
-        </div>  
+        </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">🖥️ Programmer 🖥️</p>
+            <h5>Smridhi Madan</h5>
+            <p>Smridhi is a new member of the team and is a programmer.</p>
+        </div>
+
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">🔧 Mechanic 🔧</p>
+            <h5>Will Patmore</h5>
+            <p>(Description in progress)</p>
```

Comment:
Hmm, can we just add a basic "Will is a mechanic" description like the other people have? This looks a tad unprofessional